image-builder-cli.spec

```diff
@@ -0,0 +1,97 @@
+# Do not build with tests by default
+# Pass --with tests to rpmbuild to override
+%bcond_with tests
+%bcond_with relax_requires
+
+# The minimum required osbuild version
+%global min_osbuild_version 129
+
+%global goipath         github.com/osbuild/image-builder-cli
+
+Version:        0
+
+%gometa
+
+%global common_description %{expand:
+A service for building customized OS artifacts, such as VM images and OSTree
+commits, that uses osbuild under the hood. Besides building images for local
+usage, it can also upload images directly to cloud.
+
+It is compatible with composer-cli and cockpit-composer clients.
```

Comment:
This was probably not changed when copied over from osbuild-composer. We should update it.